modules/ROOT/pages/configuration/files/file_sharing_configuration.adoc

```diff
@@ -3,6 +3,8 @@ File Sharing
 
 The sharing policy is configured on the Admin page in the *"Sharing"* section.
 
+NOTE: If you don't see the sharing section, disable your adblock browser plugin.
```

Comment:
why write a long sentence if you can tell the same information in a short one?

Reply:
because it might not be related to the adblocker. there is no 100% guarantee that the adblocker is the cause. you have to think from the users pov and if the adblocker was not the case the sentence is wrong. the onger version is an invite to take a look while the short version is a directive

Reply:
I'd write that sentence a little differently, @mmattel, but it's an excellent recommendation.

Reply:
@mmattel do you know another reason why this could happen on a fresh installed instance?

Reply:
Heaven knows 😄 

![image](https://user-images.githubusercontent.com/3321281/46014528-e35baf00-c0cf-11e8-8021-7b91651b415c.png)

Reply:
This sentence is even longer then @mmattel s :D

I am not suggesting we don't document stuff anymore, but I am also thinking about not to overwhelm the modern reader. If we write clear instruction in the format "If problem do this" it would be easier to read and to understand.

Last word has @settermjd as usual, so should I implement your suggestion?

Reply:
I understand your point, @voroyam, but I believe that the instructions are the right ones for this case.